cmd/nodecmd/status.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"github.com/ava-labs/avalanche-cli/cmd/subnetcmd"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/spf13/cobra"
+)
+
+func newStatusCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status [subnetName]",
+		Short: "(ALPHA Warning) Get node bootstrap status",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
+
+The node status command gets the bootstrap status of a node with the Primary Network. 
+To get the bootstrap status of a node with a Subnet, use --subnet flag`,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE:         statusSubnet,
+	}
+	cmd.Flags().StringVar(&subnetName, "subnet", "", "specify the subnet the node is syncing with")
+
+	return cmd
+}
+
+func statusSubnet(_ *cobra.Command, args []string) error {
+	clusterName := args[0]
+	err := setupAnsible()
+	if err != nil {
+		return err
+	}
+	if subnetName != "" {
+		_, err = subnetcmd.ValidateSubnetNameAndGetChains([]string{subnetName})
+		if err != nil {
+			return err
+		}
+		sc, err := app.LoadSidecar(subnetName)
+		if err != nil {
+			return err
+		}
+		blockchainID := sc.Networks[models.Fuji.String()].BlockchainID
+		if blockchainID == ids.Empty {
+			return ErrNoBlockchainID
+		}
+		_, err = getNodeSubnetSyncStatus(blockchainID.String(), clusterName, true)
```

Comment:
do we want to return err of print a ux user message? I think the second option cc @dasconnor